internal/xds/translator/cluster.go

```diff
@@ -695,9 +695,10 @@ func buildBackandConnectionBufferLimitBytes(bc *ir.BackendConnection) *wrappers.
 }
 
 type ExtraArgs struct {
-	metrics       *ir.Metrics
```

Comment:
same as my other comment. I don't think you need to pass in the circuit breaker via extra args.

Reply:
can you revert this whole file? seems like the formatting is off. I would expect no diff here.